Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml

```diff
@@ -0,0 +1,91 @@
+Parser:
+  Title: Process Create Event ASIM parser for Sysmon
+  Version: '0.4.1'
+  LastUpdated: June 3, 2024
+Product:
+  Name: Sysmon
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.0'
```

Comment:
ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

Reply:
DONE